src/vs/base/browser/markdownRenderer.ts

```diff
@@ -306,7 +309,9 @@ export function renderMarkdown(markdown: IMarkdownString, options: MarkdownRende
 			}
 		});
 
-	element.innerHTML = sanitizeRenderedMarkdown(markdown, markdownHtmlDoc.body.innerHTML) as unknown as string;
+	const html = markdownHtmlDoc.body.innerHTML;
+	const trustedhtml = ttPolicy ? ttPolicy.createHTML(html) : html;
```

Comment:
Given the fact that we already sanitize the markdown in line 267, i think we can remove the second level of sanitization.

However I could be completely wrong, hence feel free to close this issue.

Reply:
Possibly but if anything I'd prefer removing the first sanitize call. This is the final pass so if we remove it there's a risk that the post processing steps that happen after the first sanitize call but before this one could introduce a security issue